platform/ios/platform/darwin/test/MGLCoordinateFormatterTests.m

```diff
@@ -24,7 +24,11 @@ - (void)testStrings {
     coordinate = CLLocationCoordinate2DMake(38.9131982, -77.0325453144239);
     XCTAssertEqualObjects([shortFormatter stringFromCoordinate:coordinate], @"38°54′48″N, 77°1′57″W");
     XCTAssertEqualObjects([mediumFormatter stringFromCoordinate:coordinate], @"38°54′48″ north, 77°1′57″ west");
+    #if TARGET_OS_IPHONE || TARGET_IPHONE_SIMULATOR
     XCTAssertEqualObjects([longFormatter stringFromCoordinate:coordinate], @"38 degree(s), 54 minute(s), and 48 second(s) north by 77 degree(s), 1 minute(s), and 57 second(s) west");
+    #else
+    XCTAssertEqualObjects([longFormatter stringFromCoordinate:coordinate], @"38 degrees, 54 minutes, and 48 seconds north by 77 degrees, 1 minute, and 57 seconds west");
+    #endif
```

Comment:
Just for my own understanding - this is because MacOS localizes coords differently from iOS or is that Maplibre itself that does this?